aptos-move/move-examples/veiled_coin/sources/veiled_coin.move

```diff
@@ -237,7 +247,7 @@ module veiled_coin::veiled_coin {
 
     /// Initializes a so-called "resource" account which will maintain a `coin::CoinStore<T>` resource for all `Coin<T>`'s
     /// that have been converted into a `VeiledCoin<T>`.
-    fun init_module(deployer: &signer) {
+    fun init_module<CoinType>(resource_account: &signer) {
```

Comment:
I don't think `init_module` can be generic. Plus, it's not in line with the functionality we want: we want the `veiled_coin` module to be deployed **once** for **all** coin types.

Reply:
Correct. Changed it back to be unparameterized